src/runtime/nuxt-image-mixins.ts

```diff
@@ -144,15 +145,27 @@ export default {
     }
   },
   methods: {
+    getPlaceholder () {
+      try {
+        return this.$img.lqip(this.src)
+      } catch (e) {
+        this.error = e.message
```

Comment:
- We may unify to `onError` method
- We are hiding stack-trace, would be nice to also `console.error` original error